src/app/applications/components/applications-list/applications-list.tsx

```diff
@@ -87,34 +86,30 @@ export class ApplicationsList extends React.Component<RouteComponentProps<{}>, {
                         <div className='argo-table-list argo-table-list--clickable'>
```

Comment:
Mockup assumes screen have two apps in a row. I would try to use https://foundation.zurb.com/sites/docs/grid.html#fluid-row to archive it